ams/server/src/main/java/com/netease/arctic/server/dashboard/model/BaseMajorCompactRecord.java

```diff
@@ -16,23 +16,20 @@
  * limitations under the License.
  */
 
-package com.netease.arctic.ams.server.model;
+package com.netease.arctic.server.dashboard.model;
 
-import com.netease.arctic.ams.api.OptimizeType;
+import com.netease.arctic.server.optimizing.OptimizingType;
 import com.netease.arctic.table.TableIdentifier;
 
-// TODO remove after web changed in 0.3.1
 public class BaseMajorCompactRecord {
```

Comment:
Rename to `CompactRecord` or `OptimizedRecord`?

Reply:
I think `OptimizedRecord` is appropriate.